regulations/views/comment.py

```diff
@@ -34,6 +35,15 @@ def upload_proxy(request):
     })
 
 
+class PrepareCommentView(TemplateView):
+    template_name = 'regulations/comment.html'
+
+    def get_context_data(self, doc_number, **kwargs):
+        context = super(PrepareCommentView, self).get_context_data(**kwargs)
+        context['doc_number'] = doc_number
```

Comment:
I don't think this is required. I think the default `get_context_data` passed URL params through to the template.